sources/packages/backend/libs/sims-db/src/entities/disbursement-values.model.ts

```diff
@@ -100,4 +101,35 @@ export class DisbursementValue extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   disbursementSchedule: DisbursementSchedule;
+  /**
+   * Amount that is subtracted because of a restriction.
+   */
+  @Column({
+    name: "restriction_amount_subtracted",
+    type: "numeric",
+    nullable: true,
+    transformer: numericTransformer,
+  })
+  restrictionAmountSubtracted?: number;
+  /**
+   * References the restriction related to the disbursement
+   * due to which the amount for subtracted.
```

Comment:
The "for" here seems lost, is it supposed to be "due to which the amount was subtracted"?

Reply:
updating to `* References the restriction related to the disbursement
   * due to which the amount was subtracted.`

Reply:
Looks good, just remove the additional white spaces in the middle of the proposed sentence.